protobuf/riak-protobuf.cabal

```diff
@@ -1,5 +1,5 @@
 name:                riak-protobuf
-version:             1.0.0
+version:             0.25.0.0
```

Comment:
Seems I never published a `1.0.0` version to [hackage](https://hackage.haskell.org/package/riak-protobuf) 
Reverting to the previous numbering scheme.